cms/static/js/views/modals/course_outline_modals.js

```diff
@@ -307,14 +319,18 @@ define(['jquery', 'backbone', 'underscore', 'gettext', 'js/views/baseview',
             if (!this.$('#id_timed_examination').is(':checked')) {
                 this.$('#id_exam_proctoring').attr('disabled','disabled');
                 this.$('#id_time_limit').attr('disabled', 'disabled');
+                this.$('#id_practice_exam').attr('disabled', 'disabled');
             }
         },
         isExamTimeEnabled: function () {
             return this.$('#id_timed_examination').is(':checked');
         },
+        isPracticeExam: function () {
+            return this.$('#id_practice_exam').is(':checked');
+        },
         isValidTimeLimit: function(time_limit) {
             var pattern = new RegExp('^([0-9]|0[0-9]|1[0-9]|2[0-3]):[0-5][0-9]$');
-            return pattern.test(time_limit);
+            return pattern.test(time_limit) && time_limit !== "00:00";
```

Comment:
to be a valid time limit, the format would have to match the regex and not be 00:00